### PR TITLE
Remove parcel in code

### PR DESCRIPTION
**Pull request by sgkim126**
It closes https://github.com/CodeChain-io/codechain/issues/1818.

### CODE REVIEW

File: core/src/blockchain/body_db.rs [hunk 1]
```diff
@@ -36,27 +36,27 @@ const BODY_CACHE_SIZE: usize = 1000;
...
-    transaction_address_cache: Mutex<HashMap<H256, TransactionAddresses>>,
-    pending_transaction_addresses: Mutex<HashMap<H256, Option<TransactionAddresses>>>,
+    tx_address_by_tracker_cache: Mutex<HashMap<H256, TransactionAddresses>>,
+    pending_tx_addresses_by_tracker: Mutex<HashMap<H256, Option<TransactionAddresses>>>,
 
     db: Arc<dyn KeyValueDB>,
 }
 
-type TransactionHashAndAddress = (H256, TransactionAddresses);
+type TransactionTrackerAndAddress = (H256, TransactionAddresses);
```
Comment (majecty) [inline core/src/blockchain/body_db.rs]: How about giving the name "tracker" to H256?
Like `type Tracker = H256`

Comment (majecty) [reply core/src/blockchain/body_db.rs]: I found the third commit. I'll remove this comment.

**Comment by sgkim126** [comment]
@majecty I'm still work in progress. I'll request a review when I finish.

**Comment by sgkim126** [comment]
@majecty I'll make another PR for https://github.com/CodeChain-io/codechain/pull/1823/commits/908bdc90fcd37551a2e3551a0f782971fd264939 first.

**Comment by sgkim126** [comment]
@majecty We should decide whether the log titles are interfaces or not. If they are part of interfaces, we should raise the major version since we change "own_parcel" and "external_parcel" to "own_tx" and "external_tx" in https://github.com/CodeChain-io/codechain/pull/1823/commits/db7016a90d7c6be07dfffcbd7dc41fbe6be03566.

**Comment by majecty** [comment]
@sgkim126 I think that it is not an interface.
@kseo @foriequal0 @Remagpie @HoOngEe @joojis Wat do you think about it?

**Comment by Remagpie** [comment]
I can't remember any documents/applications/tools we made that depend on the content of CodeChain's log.
I think we don't have to treat it as an interface in that case.

**Comment by kseo** [comment]
@sgkim126 I think it's okay to change log formats now if no existing tools depend on them.

**Comment by sgkim126** [comment]
@majecty I resolved conflicts.

**Pull request merged**